app/presenters/two_factor_login_options_presenter.rb

```diff
@@ -88,12 +88,15 @@ def account_reset_url(locale:)
   end
 
   def account_reset_cancel_link
-    t(
-      'two_factor_authentication.account_reset.pending_html',
-      cancel_link: @view.link_to(
-        t('two_factor_authentication.account_reset.cancel_link'),
-        account_reset_cancel_url(token: account_reset_token),
-      ),
+    safe_join(
+      [
+        t('two_factor_authentication.account_reset.pending'),
+        @view.link_to(
+          t('two_factor_authentication.account_reset.cancel_link'),
+          account_reset_cancel_url(token: account_reset_token),
+        ),
+      ],
+      ' ',
```

Comment:
The copy for was like `Some sentence. %{cancel_link}` so I figured might as well split this out into two separate i18n keys